### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by instance-id**
I made some changes to match up functions on Disgords end

### CODE REVIEW

File: disgordrouter/context.go [hunk 1]
```diff
@@ -43,13 +43,14 @@ func (c *Context) Get(key string) interface{} {
 
 // Reply replies to the sender with the given message
 func (c *Context) Reply(args ...interface{}) (*disgord.Message, error) {
-	return c.Ses.SendMsgString(c.Msg.ChannelID, fmt.Sprint(args...))
+	Message, Error := c.Ses.SendMsg(c.Msg.ChannelID, fmt.Sprint(args...))
+	return Message, Error
```
Comment (Necroforger) [inline disgordrouter/context.go]: Simply `return c.Ses.SendMsg(c.Msg.ChannelID, fmt.Sprint(args...))` here.
It will return both values of the function

Comment (instance-id) [reply disgordrouter/context.go]: I thought it was supposed to also, for some reason though it kept saying it wasn't returning enough arguments that way when I was checking it. I will try it again. I am new to Go and still learning, so maybe it was something else that it was complaining about and I misunderstood.

File: disgordrouter/context.go [hunk 2]
```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"sync"
 
-	"github.com/andersfylling/snowflake"
+	"github.com/andersfylling/snowflake/v3"
```
Comment (Necroforger) [inline disgordrouter/context.go]: `Anders  Today at 9:43 AM`
```
remove andersfylling/snowflake
and use disgord.Snowflake
then you might be able to get around the module issue. not sure tho
the PR looks good. I'm just not officially supporting anything that is not a module
```

Comment (MostHated) [reply disgordrouter/context.go]: These are my first PR I have done. Do I have to redo it after I made the change and commit it to my fork?

Comment (Necroforger) [reply disgordrouter/context.go]: No, you can keep updating your branch until I merge it

Comment (andersfylling) [reply disgordrouter/context.go]: > These are my first PR I have done. Do I have to redo it after I made the change and commit it to my fork?

Nope. It get's automatically added to this PR. You should see a new change below this conversation: "Changed to use disgord.snowflake"

**Comment by Necroforger** [comment]
Looks good, thanks for the PR.

**Pull request merged**